acord/models/guild.py

```diff
@@ -41,7 +49,7 @@ class Guild(pydantic.BaseModel, Hashable):
 
     channels: List[Any]  # This gets sorted out with validators
     """ All channels in the guild """
-    default_message_notifications: Literal[0, 1]
+    default_message_notifications: GuildMessageNotification
```

Comment:
:class:\`GuildMessageNotification\`

Reply:
![image](https://user-images.githubusercontent.com/77537491/146684146-a0a0f0b4-43b0-4d66-8f43-278753a919ad.png)
 is this what you mean?

Reply:
remove the double \`, you only need single ones

Reply:
okithen

Reply:
Fixed in https://github.com/Mecha-Karen/acord/pull/11/commits/b8f6ab529a1024a69e5cbc142934dbb22ba31b2c